gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -334,86 +496,351 @@ def apply_stratified_filters(filtering_qc_metrics: List[str]) -> hl.Table:
 
 
 def apply_regressed_filters(
+        sample_qc_ht: hl.Table,
         filtering_qc_metrics: List[str],
         include_unreleasable_samples: bool,
+        n_pcs: int = 16,
 ) -> hl.Table:
-    sample_qc_ht = get_sample_qc('bi_allelic').ht()
+    """
+    Computes sample QC metrics residuals after regressing out population PCs and determines what samples are outliers
+    that should be filtered.
+
+    .. note::
+
+        Default left and right MAD for `compute_stratified_metrics_filter` is 4. We modify this for n_singleton_residual
+        to be (math.inf, 8.0) and for r_het_hom_var_residual to be (math.inf, 4.0).
```

Comment:
I think because it is the number of median absolute deviation from the mean using this is basically saying we don't want to have a lower cutoff for the metric, but we do want an upper cutoff

https://github.com/broadinstitute/gnomad_methods/blob/19dad9327a5520ee530037b6770ad4924f905f7d/gnomad/sample_qc/filtering.py#L147